wake-lock/wakelock-applicability-manual.https.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>wake lock applicability test</title>
+<link rel="help" href="https://w3c.github.io/wake-lock/#dfn-the-wake-lock-is-applicable">
+<meta name="flags" content="interact">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<p>
+  Lock and turn off the screen, then turn it on and unlock the screen in a minute.
```

Comment:
This is unclear, are you asking the tested to wait 60 seconds?

Reply:
I theory, the tests will run by the time the tested turns the screen off and then on again. So likely they only have to wait maybe 1 or 2 seconds, right?

Reply:
I didn't mean waiting, I just wanted to make sure the interaction should be complete within 60 seconds, before test timeout. Anyway, I will improve the description.